src/repositories/projectRepository.ts

```diff
@@ -289,3 +305,20 @@ export const makeProjectListed = async (id: number): Promise<void> => {
     .updateEntity(true)
     .execute();
 };
+
+export const findProjectsBySlugArray = async (
+  slugArray: string[],
+): Promise<Project[]> => {
+  // TODO should refactor it and convert it to single query
+  const projects: Project[] = [];
+  const result = await Promise.all(
+    slugArray.map(slug => findProjectBySlug(slug)),
+  );
```

Comment:
For this we can us the ```IN```operator in sql.
project.slug IN (:...slugs)

But will need to investigate how to do it with the slughistory part.